services/QuillLMS/app/services/vitally_integration/serialize_vitally_sales_account.rb

```diff
@@ -129,6 +133,14 @@ def data
       "https://www.quill.org/cms/schools/#{@school.id}"
     end
 
+    private def first_subscription
+      @school&.subscriptions&.order(start_date: :asc)&.first
+    end
+
+    private def earliest_premium_start_date
+      first_subscription&.start_date || NOT_APPLICABLE
+    end
```

Comment:
Purely stylistic, and you can skip this suggestion, but these could be one-liners:
```ruby
private def first_subscription = @school&.subscriptions&.order(start_date: :asc)&.first
private def earliest_premium_start_date = first_subscription&.start_date || NOT_APPLICABLE
```